README.md

```diff
@@ -73,3 +73,13 @@ used to treat these warnings as test failures.
 There are also some combinations of parametrization values which will always be skipped (because
 they are combinations of features which are not supported). These are nothing to worry about
 either - it's just that using `pytest.skip` was the least bad way of omitting these cases.
+
+## Troubleshooting
+
+If some tests are failing because of the mock harvest source not matching then it's likely that 
+the `vars.conf` was not updated in nginx in the `static-mock-harvest-source` container. 
+Run these commands to fix it-
+
+  docker exec -it static-mock-harvest-source bash
+  echo $'\nmap $host $mock_absolute_root_url { default "http://static-mock-harvest-source:11088/"; }' >> /etc/nginx/vars.conf
+  service nginx reload
```

Comment:
Problem is this fix won't be persistent done from within the docker container.

Reply:
This is just a quick run so that they won't have to stop the docker container and run bootstrap.sh, I'll add some notes for that.